src/util.js

```diff
@@ -6,23 +6,14 @@
 plugin.service('wgnUtil', [
 	function() {
 
-		/**
-		 * Service
-		 *
-		 * @type {Object}
-		 */
-		var Service = this;
```

Comment:
@JoeBello hey man I'm actually gonna fight ya on this one... it's 100% a best practice to not use `this` directly in angular services, controllers, etc.  You really want to be doing:
```js
plugin.service('wgnUtil', [function () {
  var srv = this;

  srv.foo = 'whatever';

  srv.bar = function (param) { alert() };

  return srv;
}]);
```

Reply:
I can't argue this one.